storage/mysql/queue.go

```diff
@@ -138,20 +138,32 @@ UPDATE
 }
 
 func (s *MySQLStorage) RetrieveNextCommand(r *mdm.Request, skipNotNow bool) (*mdm.Command, error) {
```

Comment:
minor but if `r.status IS NULL` is always part of the query then it can move to the main query and only condition on  `!skipNotNow` to append `OR r.status = 'NotNow'` to simplify code and avoid reassigning of `statusWhere`

Reply:
but this could introduce more changes and risk so yea ...

Reply:
another option is to pass `skipNotNow` in as a parameter, and perform boolean operations on it. This would get rid of the need to append at all.